### DISCUSSION TITLE
One-time event

### DISCUSSION THREAD

**Discussion by Celx2** [iSoron/uhabits#1579] [Feature Requests]
Hi.

I'm not sure if it's currently possible to add a one-time thing task.

I mean I add the task and it will appear every day until I get it done once and then dissapear completely.

Would something like that be possible?

Regards

**Comment by Almtesh** [comment]
If you must do something only once, I suggest you to use a task manager like OpenTasks.

**Comment by iSoron** [comment]
Thanks for the feature request, @Celx2. Unfortunately that's not currently possible. I'm also inclined to think that this feature is outside the scope of the app; I would suggest other apps for handling one-time/scheduled events.
